x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiBasicTable, EuiButtonIcon, EuiCopy } from '@elastic/eui';
+import { useActions, useValues } from 'kea';
+import { CriteriaWithPagination } from '@elastic/eui/src/components/basic_table/basic_table';
```

Comment:
nit - may as well keep EUI imports together

```suggestion
import { EuiBasicTable, EuiButtonIcon, EuiCopy } from '@elastic/eui';
import { CriteriaWithPagination } from '@elastic/eui/src/components/basic_table/basic_table';
import { useActions, useValues } from 'kea';
```

Also weird/interesting that EUI doesn't export CriteriaWithPagination directly from the main module 🤔

Reply:
It does, I think I relied on an auto import from VS Code.

Reply:
Just kidding, no it doesn't

Reply:
a30f8d5